packages/color-helpers/package.json

```diff
@@ -37,23 +37,21 @@
 		"README.md",
 		"dist"
 	],
-	"eslintConfig": {
-		"rules": {
-			"@typescript-eslint/no-loss-of-precision": "off"
-		}
+	"devDependencies": {
+		"node-fetch": "^3.2.4"
```

Comment:
Should we switch to node 18 as the main version?
This has `fetch` natively.

Reply:
Doing that works great but logs this: 

```
 node scripts/check-changes.mjs
(node:14076) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
No changes detected
```

Should we [live on the edge](https://www.youtube.com/watch?v=7nqcL0mjMjw) here? 

FWIW I'm OK with the warning :)

Reply:
Maybe not :D

We can add an issue to switch to native node fetch whenever it no longer is experimental. Don't want to block development if there ever is an API change :)